core.Dockerfile

```diff
@@ -0,0 +1,88 @@
+ARG           BUILD_BASE="ubuntu:focal"
+ARG         DEPENDENCIES="dependencies"
+ARG              RUNTIME="runtime"
+ARG          DATAFED_DIR="/datafed"
+ARG DATAFED_INSTALL_PATH="$DATAFED_DIR/install"
+ARG            GCS_IMAGE="code.ornl.gov:4567/dlsw/datafed/gcs-ubuntu-focal"
+ARG            BUILD_DIR="$DATAFED_DIR/source"
+ARG              NVM_DIR="$DATAFED_DIR/.nvm"
+ARG              NVM_INC="$DATAFED_DIR/.nvm/versions/node/v13.14.0/include/node"
+ARG              NVM_BIN="$DATAFED_DIR/.nvm/versions/node/v13.14.0/bin"
+ARG              LIB_DIR="/usr/local/lib"
+
+FROM ${DEPENDENCIES} AS core-build
+
+ARG DATAFED_DIR
+ARG BUILD_DIR
+ARG DATAFED_INSTALL_PATH
+
+# For communicating with repo server
+EXPOSE 7512
+# For listening to web server
+EXPOSE 7513
+# ArangoDB port
+EXPOSE 8529
```

Comment:
I don't think this is needed, as ArangoDB will be running in it's own container. If this was in the original Dockerfile it was an error. ArangoDB binds to this port not the core server.